.github/workflows/android.yml

```diff
@@ -0,0 +1,48 @@
+name: Build APK
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: 'zulu'
```

Comment:
this line isn't necessary either

Reply:
I can remove it if you want I just like zulu best

Reply:
If there are advantages for using Zulu (faster builds or anything else) we can keep it, else remove

Reply:
Turns out in v2 distribution is required and v1 used zulu by default anyways